.gitignore

```diff
@@ -1,3 +1,4 @@
 *.pyc
 __pycache__
-env
\ No newline at end of file
+env
+00*.py
```

Comment:
Refer to [this SO comment](https://stackoverflow.com/questions/28035119/should-i-be-adding-the-django-migration-files-in-the-gitignore-file#comment47660873_28035246) on why to put migrations in gitignore when the project is under development.

Reply:
Thanks @apb7.
That is very helpful.